src/terminal/parser/OutputStateMachineEngine.hpp

```diff
@@ -131,7 +131,11 @@ namespace Microsoft::Console::VirtualTerminal
             DECREQTPARM_RequestTerminalParameters = VTID("x"),
             DECSCUSR_SetCursorStyle = VTID(" q"),
             DECSTR_SoftReset = VTID("!p"),
-            DECSCPP_SetColumnsPerPage = VTID("$|")
+            DECSCPP_SetColumnsPerPage = VTID("$|"),
+            XT_PushSgr = VTID("#{"),
+            XT_PushSgrAlias = VTID("#p"),
+            XT_PopSgr = VTID("#}"),
+            XT_PopSgrAlias = VTID("#q"),

```

Comment:
Nit: I've been trying to keep these in some sort of order, so they're sorted alphabetically by their intermediate/final chars. Ideally the new ones would come before `DECSCPP` (`#` before `$`), and in the order `p`, `q`, `{` , `}` (assuming we're keeping the aliases).

Reply:
Thank you--sorted. I do insist on keeping the aliases--without them, format strings with VT sequences in languages like C# become way more cumbersome (imagine if `%` were a common VT sequence char).